docs/src/pages/components/timeline/AlternateTimeline.js

```diff
@@ -14,27 +15,35 @@ export default function AlternateTimeline() {
           <TimelineDot />
           <TimelineConnector />
         </TimelineSeparator>
-        <TimelineContent>Eat</TimelineContent>
+        <TimelineContent>
+          <Typography>Eat</Typography>
```

Comment:
What do you think about using the `Typography` component right inside `TimelineContent`? It feels like the most common use case is to render a string directly, I believe we can save the intermediary component.

```diff
diff --git a/packages/material-ui-lab/src/TimelineContent/TimelineContent.js b/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
index 2f117df81..fbb6c80da 100644
--- a/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
+++ b/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
@@ -25,7 +25,8 @@ const TimelineContent = React.forwardRef(function TimelineContent(props, ref) {
   const { classes: contextClasses = {} } = React.useContext(TimelineItemContext);

   return (
-    <div
+    <Typography
+      component="div"
       className={clsx(
         classes.root,
         contextClasses.content,
```

```suggestion
          Eat
```

Reply:
Does this mean that we will need to extend the props to contain the `Typography` props?

Reply:
Yes, it would also mean that more props would be available. If you don't like that, we can simply do:

```diff
diff --git a/packages/material-ui-lab/src/TimelineContent/TimelineContent.js b/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
index 2f117df81..bd9a45202 100644
--- a/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
+++ b/packages/material-ui-lab/src/TimelineContent/TimelineContent.js
@@ -6,9 +6,10 @@ import { withStyles } from '@material-ui/core/styles';
 import TimelineContext from '../Timeline/TimelineContext';
 import TimelineItemContext from '../TimelineItem/TimelineItemContext';

-export const styles = () => ({
+export const styles = (theme) => ({
   /* Styles applied to the root element. */
   root: {
+    ...theme.typography.body1,
     flex: 1,
     padding: '6px 16px',
   },
```

Reply:
I'll extends the props there too, would be convinient to just use `color="textSecondary"` as an example on the `TimelineContent`/`TimelineOppositeContent`